generator/generator-opc-generator/generators/app/templates/test/_test.js

```diff
@@ -0,0 +1,8 @@
+import { html, fixture, expect } from '@open-wc/testing';
+
+describe('my-test', () => {
+  it('works', async () => {
+    const el = await fixture(html`<<%= componentName %> name="<%= authorName %>"></<%= componentName %>>`);
+    expect(el.count).exist();
+  });
```

Comment:
@hybridx add a comment on line no  `Add more tests here`

Reply:
Changed the test cases to jest, as it is much faster and easier to work with